packages/workflow/src/TypeValidation.ts

```diff
@@ -148,6 +153,96 @@ export const tryToParseObject = (value: unknown): object => {
 	}
 };
 
+const ALLOWED_FORM_FIELDS_KEYS = [
+	'fieldLabel',
+	'fieldType',
+	'placeholder',
+	'fieldOptions',
+	'multiselect',
+	'multipleFiles',
+	'acceptFileTypes',
+	'formatDate',
+	'requiredField',
+];
+
+const ALLOWED_FIELD_TYPES = [
+	'date',
+	'dropdown',
+	'email',
+	'file',
+	'number',
+	'password',
+	'text',
+	'textarea',
+];
+
+export const tryToParseFormFields = (value: unknown): FormFieldsParameter => {
```

Comment:
I suggest including json in the name to make it clear what we are parsing from
```suggestion
export const tryToParseJsonToFormFields = (value: unknown): FormFieldsParameter => {
```